perf/benchmark/runner/runner.py

```diff
@@ -281,24 +289,56 @@ def kubectl_cp(from_file, to_file, container):
     run_command_sync(cmd)
 
 
+def run_nighthawk(pod, remote_cmd, labels):
+    # Use a local docker instance of Nighthawk to control nighthawk_service running in the pod
+    # and run transforms on the output we get.
+    docker_cmd = "docker run --network=host {docker_image} {remote_cmd}".format(
+        docker_image=NIGHTHAWK_DOCKER_IMAGE, remote_cmd=remote_cmd)
+    print(docker_cmd, flush=True)
+    process = subprocess.Popen(shlex.split(docker_cmd), stdout=subprocess.PIPE)
+    (output, err) = process.communicate()
+    exit_code = process.wait()
+
+    if exit_code == 0:
+        with tempfile.NamedTemporaryFile(dir='/tmp', delete=True) as tmpfile:
+            dest = tmpfile.name
+            with open("%s.json" % dest, 'wb') as f:
+                f.write(output)
+            print("Dumped Nighthawk's json to {dest}".format(dest=dest))
+
+            # Send human readable output to the command line.
+            os.system(
+                "cat {dest}.json | docker run -i {docker_image} nighthawk_output_transform --output-format human".format(docker_image=NIGHTHAWK_DOCKER_IMAGE, dest=dest))
+            # Transform to Fortio's reporting server json format
+            os.system("cat {dest}.json | docker run -i {docker_image} nighthawk_output_transform --output-format fortio > {dest}.fortio.json".format(
```

Comment:
I think for here the file name you already add `nighthawk` label in it, so`.json` is ok, why do you add `.fortio.json`?

Reply:
You are right, `.fortio.` can be left out, stripped it in 703343815cf751cecca0a35069c4bf0d33e1dacf. .

Also added a few TODOs concerning outputting more details into different files:
```
            # TODO(oschaaf): We output the global aggregated statistics here of request_to_response, which excludes connection set up time.
            # It would be nice to dump a series instead, as we have more details available in the Nighthawk json:
            # - queue/connect time
            # - initiation time to completion (spanning the complete lifetime of a request/reply, including queue/connect time)
            # - per worker output may sometimes help interpret plots that don't have a nice knee-shaped shape.

```

To support this on the Nighthawk side we would need a new option to configure the fortio output formatter (+ some small tweaks in the fortio output formatter implementation).